java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -179,11 +181,11 @@ public interface ListBaseCommands {
      * @return The length of the list after the push operations.
      * @example
      *     <pre>{@code
-     * Long pushCount1 = client.rpush("my_list", new String[] {"value1", "value2"}).get()
-     * assert pushCount1 == 2L
+     * Long pushCount1 = client.rpush("my_list", new String[] {"value1", "value2"}).get();
```

Comment:
redis returns an integer, why the return value is long?
For DEL it makes sense as we can aggregate the results over multi nodes, but why here?

Reply:
It comes from here: https://github.com/aws/glide-for-redis/blob/4681552df748b4865d0499869853115638d44405/java/src/lib.rs#L24-L25 where `Int` is `i64` https://github.com/amazon-contributing/redis-rs/blob/2a903228d3fd8b49a3984c479959142a8b8826b5/redis/src/types.rs#L314
so it is mapped to `Long`
Do you have `i32` or `u32` in other wrappers as a command return type?